### ISSUE TITLE
False positive with detected-twitter-oauth when using Twitter python lib.

### ISSUE THREAD

**Issue by ItsCalebJones** [returntocorp/sgrep-rules#2629]
**Describe the bug**
A clear and concise description of what the bug is and for which rule. Is the rule producing a false positive? Is it incorrect or missing something?

The 'detected-twitter-oauth' rule is looking for hard coded Twitter oath credentials, its being triggered in a requirements file for a Python project using a official Twitter API library.

```
 Blocking Findings:

  poetry.lock 
     generic.secrets.security.detected-twitter-oauth.detected-twitter-oauth
        Twitter OAuth detected
        Details: https://sg.run/Lwb7

       2993┆ {file = "twitter-1.18.0-py2.py3-none-any.whl", hash = "sha256:52545fd3b70d3d3807d3ce62d1a256727856d784d1630d64dedcc643aaf0b908"},
          ⋮┆----------------------------------------
       2994┆ {file = "twitter-1.18.0.tar.gz", hash = "sha256:acdc85e5beea752967bb64c63bde8b915c49a31a01db1b2fecccf9f2c1d5c44d"},
```

**To Reproduce**
Sample code to reproduce this behavior.

Create a file with the above twitter info above.

**Expected behavior**

**Priority**
How important is this to you?
- [ ] P0: blocking me from making progress
- [ ] P1: this will block me in the near future
- [X] P2: annoying but not blocking me

**Additional Context**

**Comment by LewisArdern** [comment]
Thanks for the report @ItsCalebJones we should fix this up, will internally track it and let you know when its fixed.

**Comment by LewisArdern** [comment]
We merged this into to develop should be live soon, feel free to re-open if its still an issue.
